### PR TITLE
WDT-659 cleanup CLAException for compare_model.py

### PR DESCRIPTION
**Pull request by jshum2479**
This PR cleanup CLAExceptions and exit code logic

### CODE REVIEW

File: core/src/main/python/compare_model.py [hunk 1]
```diff
@@ -254,20 +260,11 @@ def main():
 
         for f in [model1, model2]:
             if not os.path.exists(f):
-                raise CLAException("Model %s does not exists" % f)
+                raise CLAException(2, 'WLSDPLY-85717', [f])
```
Comment (rakillen) [inline core/src/main/python/compare_model.py:263]: Can this use ExitCode.ERROR ? (recently merged to main).
3 places in this file.

File: core/src/main/python/compare_model.py [hunk 2]
```diff
@@ -254,20 +260,11 @@ def main():
...
-        model2_file = JFile(model2)
-
-        if not (FileUtils.isYamlFile(model1_file) or FileUtils.isJsonFile(model1_file)):
-            raise CLAException("Model extension must be either yaml or json")
-
-        if not (FileUtils.isYamlFile(model1_file) and FileUtils.isYamlFile(model2_file)
-                or FileUtils.isJsonFile(model1_file) and FileUtils.isJsonFile(model2_file)):
-            ext = os.path.splitext(model1)[1]
-            raise CLAException("Model %s is not a %s file " % (model2, ext))
+                raise CLAException(2, 'WLSDPLY-85718', [f])
```
Comment (ddsharpe) [inline core/src/main/python/compare_model.py:265]: Shouldn't these literal 2's use the static ExitCode.ERROR

File: core/src/main/python/compare_model.py (hunk 1 above)
Comment (jshum2479) [reply core/src/main/python/compare_model.py:263]: Sure.

File: core/src/main/python/compare_model.py [hunk 3]
```diff
@@ -232,6 +232,12 @@ def debug(format_string, *arguments):
     else:
         _logger.finest(format_string, arguments)
 
+def _checkModelExtension(file):
```
Comment (ddsharpe) [inline core/src/main/python/compare_model.py:235]: python function names should be Snake case, not camel case.

Comment (jshum2479) [reply core/src/main/python/compare_model.py]: fixed

**Review by ddsharpe — APPROVED** [review]

**Review by rakillen — APPROVED** [review]

**Review by CarolynRountree — APPROVED** [review]

**Pull request merged**